routers/events.router.js

```diff
@@ -219,46 +219,35 @@ router.post("/", async (req, res, next) => {
 });
 
 // edit event by id
-router.patch(`/:eventId`, validateIds, async (req, res, next) => {
-  try {
-    const { eventId } = req.params;
-
-    const foundEvent = await Event.findById(eventId)
-      .populate({ path: `creator`, select: { password: 0, email: 0, __v: 0 } });
+router.patch(
+  `/:eventId`,
+  uploader.single("file"),
+  validateIds,
```

Comment:
why not validateIds before uploading file

Reply:
yeah sure I just changed it